google_guest_agent/network/manager.go

```diff
@@ -0,0 +1,39 @@
+package network
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/GoogleCloudPlatform/guest-agent/metadata"
+)
+
+// ManagerService is an interface for setting up network configurations
+// using different network managing services, such as systemd-networkd and wicked.
+type ManagerService interface {
+	IsManaging(ctx context.Context, iface string) (bool, error)
```

Comment:
Please add a comment/doc to each of these methods.

Reply:
Done.